s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3.scala

```diff
@@ -691,7 +691,7 @@ object S3 {
    * @param materializer materializer to run with
```

Comment:
I deprecated all the Java DSL that required a `Materializer` and used `ClassicActorSystemProvider` instead of `ActorSystem` for the new overloads.